tools/java/org/unicode/cldr/test/CheckCLDR.java

```diff
@@ -778,6 +787,9 @@ public String getMessage() {
                     String fixedApos = MessageFormat.autoQuoteApostrophe(messageFormat);
                     MessageFormat format = new MessageFormat(fixedApos);
                     message = format.format(parameters);
+                    if (errorCodesPath.contains(subtype)) {
+                        message += "; see <a href='http://cldr.unicode.org/translation/error-codes#" + subtype.name() + "'  target='cldr_error_codes'>" + subtype + "</a>.";
```

Comment:
Add link to new page for certain subtypes. We can add more info over time to http://cldr.unicode.org/translation/error-codes